enrichment-web/src/main/java/eu/europeana/enrichment/web/controller/NERController.java

```diff
@@ -78,31 +79,39 @@ public ResponseEntity<String> createNamedEntitiesStory(
 			@RequestParam(value = "linking", required = false) String linking,
 			@RequestParam(value = "nerTools", required = false) String nerTools,
 			@RequestParam(value = "original", required = false, defaultValue = "false") Boolean original,
+			@RequestParam(value = "force", required = false,defaultValue = "false") Boolean force,
 			HttpServletRequest request) throws Exception, HttpException, SolrServiceException {
 	
 		verifyWriteAccess(Operations.CREATE, request);
 		
 		if(translationTool==null) translationTool=EnrichmentConstants.defaultTranslationTool;
 		if(property==null) property=EnrichmentConstants.STORY_ITEM_DESCRIPTION;
-		if(linking==null) linking=EnrichmentConstants.defaultLinkingTool;
-		if(nerTools==null) nerTools=EnrichmentConstants.dbpediaSpotlightName + "," + EnrichmentConstants.stanfordNer;
+		if(linking==null) linking=EnrichmentConstants.WIKIDATA_LINKING;
+		if(nerTools==null) nerTools=NerTools.Dbpedia.getStringValue() + "," + NerTools.Stanford.getStringValue();
 		if(original==null) original=false;
+		if(force==null) force=false;
 		
 		List<String> linkingList=new ArrayList<>(Arrays.asList(HelperFunctions.toArray(linking,",")));
 		List<String> nerToolsList=new ArrayList<>(Arrays.asList(HelperFunctions.toArray(nerTools,",")));
 		validateTranslationParams(storyId, null, translationTool, property, false);
 		validateNERTools(nerToolsList);
 		validateNERLinking(linkingList);
 		
-		List<NamedEntityImpl> result = enrichmentNerService.getEntities(storyId, null, property, nerToolsList);
 		String resultJsonLd = null;
-		if(!result.isEmpty()) {
-			resultJsonLd="{\"Result\" : \"NamedEntity-s have been already crrated! Please use get method to get them!\"}";
-		}
-		else {	
-			enrichmentNerService.createNamedEntitiesForStory(storyId, property, nerToolsList, true, linkingList, translationTool, original, false);
+		if(force) {

```

Comment:
this should overwrite the previously generated data, which means the that previously generated enrichments should be deleted